### PR TITLE
Ft cpdt 87 - Banner

### PR DESCRIPTION
**Pull request by stevenmwade**
Banner and nav components.

### CODE REVIEW

File: src/components/BBanner/BBanner.vue [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+<script>
+import BNav from '../BNav'
+
+export default {
+  name: 'b-banner',
+  props: {
+    primary: Boolean,
+    secondary: Boolean,
+    tirtiary: Boolean
```
Comment (fobesjason) [inline src/components/BBanner/BBanner.vue]: tertiary

Comment (stevenmwade) [reply src/components/BBanner/BBanner.vue]: VSCode says it has spell check. Lies.

Comment (eleanormal) [reply src/components/BBanner/BBanner.vue]: Are these exclusive booleans like only one can be true? Is there a way we can enforce that?

Comment (stevenmwade) [reply src/components/BBanner/BBanner.vue]: You can't check other props. You can do that in the computed class function.

Comment (eleanormal) [reply src/components/BBanner/BBanner.vue]: Yeah we should do that because presumably it's not valid for it to have all 3 classes

**Comment by eleanormal** [comment]
This story got pulled out of the sprint due to redesign.

**Pull request closed without merging**